vueapp/src/components/Player.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+<v-list-item>
+  <template v-slot:prepend>
+    <v-icon
+      :icon="player.isGuest ? 'mdi-account-box-outline' : 'mdi-account-circle'"
+    ></v-icon>
+  </template>
+  <v-list-item-title>
+    {{ player.name }}
+  </v-list-item-title>
+  <template v-slot:append>
+    <slot name="button"></slot>
```

Comment:
This slot may be for more than just buttons... Icon indicators, etc.   Maybe a more generic name?

Reply:
> This slot may be for more than just buttons... Icon indicators, etc. Maybe a more generic name?

Renamed to `append` to match slot in `v-list-item`.